### PR TITLE
Public feed - New servlet class and datastore logic

### PR DESCRIPTION
**Pull request by MCharming98**
Finished: #9, Add MessageFeedServlet class;  #10, Add fetch all messages logic to DataStore class
Update: Also #11 #12, modified eslint configuration

### CODE REVIEW

**Comment by maksymko** [comment]
@MCharming98 Chen, please add some of your teammates as reviewers (you don't have to add me).

File: src/main/webapp/js/public-feed-loader.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const messageDiv = document.createElement('div');
+  messageDiv.classList.add("message-div");
+  messageDiv.appendChild(headerDiv);
+  messageDiv.appendChild(bodyDiv);
+
+  return messageDiv;
+}
+  
+// Fetch data and populate the UI of the page.
+function buildFeedUI(){
```
Comment (maksymko) [inline src/main/webapp/js/public-feed-loader.js:65]: I think lint error here is a configuration issue. I've changed linter configuration, which hopefully should fix this problem. Just make sure to pull the change from master.

Comment (MCharming98) [reply src/main/webapp/js/public-feed-loader.js:65]: I pulled the changes to the config file but the problem is still not solved

**Comment by alyssa-lau** [comment]
This looks good to me

**Comment by alyssa-lau** [comment]
This looks good to me.

**Review by alyssa-lau — APPROVED** [review]
This looks good to me.

**Pull request merged**